Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM elifesciences/php_cli:d83fc4714914898b0842199578e1cc88d9feab2a
+ARG image_tag=latest
+FROM elifesciences/api-dummy_composer:${image_tag} AS build
+FROM elifesciences/php_7.0_cli:656bb4bdf1e49a5e80337e2a7c4f44f10c3f52b0
 
-USER elife
 ENV PROJECT_FOLDER=/srv/api-dummy
-RUN mkdir ${PROJECT_FOLDER}
+
+USER root
```

Comment:
why changing to `root`? Seeing no `RUN` commands that require it after this point

Reply:
Assuming the user's inherited from `php_7.0_cli`, it's `www-data` at this point and the `WORKDIR` will create `/srv/api-dummy` (though should be owned by `elife`).

Reply:
Yes, originally it changed to `elife` which has just enough permissions